src/main/java/com/agiletec/plugins/jpversioning/apsadmin/global-messages_it.properties

```diff
@@ -1,7 +1,7 @@
 jpversioning.code=jpversioning
 jpversioning.name=Versionamento Contenuti
 
-jpversioning.admin.menu=Versionamento
+jpversioning.admin.menu=Versioning
```

Comment:
In this case, the Italian label "Versionamento" is correct. We should update the AppBuilder translation "Versioning" in "Versionamento" instead.

Reply:
@ichalagashvili after considering this you can close the PR when you are ready.